db.go

```diff
@@ -560,33 +666,29 @@ func (x *dummyPermitDB) GetPermits() (map[string]*Permit, error) {
 	return copy, nil
 }
 
-func (x *dummyPermitDB) SetPermit(identity string, permit *Permit) error {
+func (x *dummyPermitDB) SetPermit(userId UserId, permit *Permit) error {
 	x.permitsLock.Lock()
-	x.permits[CanonicalizeIdentity(identity)] = permit
+	x.permits[userId] = permit
 	x.permitsLock.Unlock()
 	return nil
 }
 
 func (x *dummyPermitDB) RenameIdentity(oldIdent, newIdent string) error {
-	x.permitsLock.Lock()
-	defer x.permitsLock.Unlock()
-
-	oldIdent = CanonicalizeIdentity(oldIdent)
-	newIdent = CanonicalizeIdentity(newIdent)
-
-	permit, oldExists := x.permits[oldIdent]
-	if !oldExists {
-		return ErrIdentityPermitNotFound
-	}
-	if _, newExists := x.permits[newIdent]; newExists {
-		return ErrIdentityExists
-	}
-
-	delete(x.permits, oldIdent)
-	x.permits[newIdent] = permit
-
+	// In this design, we store Authenticator and PermitDB in the same table, so we let the Authenticator portion
```

Comment:
Appropriate comment, but I think we ought to remove RenameIdentity from the PermitDB interface. I don't see how it  makes any sense now.